pubsub/natspubsub/nats.go

```diff
@@ -169,7 +169,11 @@ type TopicOptions struct{}
 
 // SubscriptionOptions sets options for constructing a *pubsub.Subscription
 // backed by NATS.
-type SubscriptionOptions struct{}
+type SubscriptionOptions struct {
+	//Queue sets the subscription as a QueueSubcription
```

Comment:
Nit: For consistency, please include a space after `//` and end comments with a `.`.

Reply:
Done!